pkg/kind/cluster.go

```diff
@@ -108,11 +110,23 @@ func (c *Cluster) getConfig() ([]byte, error) {
 		}
 	}
 
+	var registryConfig string = func() string {
```

Comment:
Does this need to be structured this way? I prefer a more simple way to set a variable value

Reply:
Nope! Agreed, this should just be simplified, I had this pattern previously and forgot to simplify it